src/System.Windows.Forms/tests/IntegrationTests/System.Windows.Forms.IntegrationTests.Common/TestHelpers.cs

```diff
@@ -316,6 +296,34 @@ public static bool PressTabsOnProcess(Process process, int times)
             return PressOnProcess(process, keys);
         }
 
+        /// <summary>
+        /// Bring the specified form to the foreground
+        /// </summary>
+        /// <param name="form">The form</param>
+        public static void BringToForeground(Form form)
+        {
+            if (form == null)
+                throw new ArgumentNullException(nameof(form));
+
+            form.WindowState = FormWindowState.Minimized;
+            form.Show();
+            form.WindowState = FormWindowState.Normal;
+        }
+
+        /// <summary>
+        /// Set the culture for the current thread
+        /// </summary>
+        /// <param name="culture">The culture</param>
+        public static void SetCulture(string culture)
```

Comment:
```suggestion
        public static void SetCulture(this Thread thread, string culture)
```

This signature will make the callsite look somewhat nicer:
```
Thread.CurrentThread.SetCulture("en-US");
```